test/karma/test-app/prerender-test/karma.spec.ts

```diff
@@ -61,6 +81,16 @@ describe('prerender', () => {
     const greenTextStyle = getComputedStyle(greenText.querySelector('text-green'));
     expect(greenTextStyle.color).toBe('rgb(0, 255, 0)');
   });
+
+  it('should render an svg child', async () => {
+    const teardown = patchConsoleError();
+    app = await setupDom('/prerender/index.html');
+    await waitForChanges(500);
+
+    const testSvg = app.querySelector('test-svg');
+    expect(testSvg.className).toBe('hydrated');
+    teardown();
```

Comment:
Will this get execute if the test fails or `throw`s somewhere between instantiating it and here??

Reply:
mmm probably not, in which case the test suite is failing _anyway_, but I can wrap the test in a `describe` so we can `afterAll`